### DISCUSSION TITLE
Issues with slug only pages

### DISCUSSION THREAD

**Discussion by patrickomeara** [statamic/cms#7104] [Q&A] [answered]
Hi all,

I have two collections Pages and Terms

**Pages** 
route: `{slug}`
eg
/tour
/support
/features

**Terms**
route: `{slug}`
eg.
/privacy
/cookies

A. Is it possible to have the same route like this?
B. Whenever the pages route is `{slug}` the non-home pages won't load the layout, only the template content, if I change the route to `pages/{slug}` the template renders, however the home page `/` gets a 404.

What's the best way to accomplish this?
Thanks in advance.

**Comment by edalzell** [comment, marked as answer]
> A. Is it possible to have the same route like this?

Yes

> B. Whenever the pages route is {slug} the non-home pages won't load the layout, only the template content, if I change the route to pages/{slug} the template renders, however the home page / gets a 404.

I suspect that you need to add "Expect Root" page on on the Pages collection.

**Reply by patrickomeara** [reply]
Thanks for your help. The "Expect Root" was the issue.
